problems/reverseString.yaml

```diff
@@ -0,0 +1,26 @@
+id: 4
+title: ReverseString
+text: |
+  Write a function called reverseString that takes a string parameter 
+  and returns the reverse version of said string.
+  
+  Refer to example below.
+  
+  reverseString(YellowBrickCode) = edoCkcirBwolleY
```

Comment:
The example should probably include `'` too.
So, `'YellowBrickCode'` instead of `YellowBrickCode`